readme.md

```diff
@@ -143,6 +143,8 @@ git clone https://github.com/nim-works/nimskull.git
 cd nimskull
 ./koch.py boot -d:release
 ./koch.py tools -d:release
+ Build all nimSkull documentation
+./koch.py docs -d: release
```

Comment:
I think that the step should be listed separately, below the "add the `bin` directory to your PATH" paragraph.